Dockerfile

```diff
@@ -14,6 +14,11 @@ WORKDIR protocol
 RUN npm install
 RUN scripts/buildContracts.sh
 
+# Environment variables to set up email notifications.
+ENV SENDGRID_API_KEY=
```

Comment:
Just curious, what's the purpose of setting them equal to nothing?

Reply:
Just mostly here as documentation, we could remove these lines if you prefer.

Reply:
Nope, just wanted to make sure it was just documentation and not something more subtle :).